src/open_inwoner/utils/ckeditor.py

```diff
@@ -100,11 +100,20 @@ def get_product_rendered_content(product):
                 icon.attrs.update(
                     {
                         "aria-hidden": "true",
-                        "aria-label": _("Opens in new window"),
                         "class": "material-icons",
                     }
                 )
                 icon.append("open_in_new")
+                # Create the screen reader only text span element
+                sr_only_text = soup.new_tag("span")
```

Comment:
Comment is unnecessary. Add a newline between the `icon` code and this part, and make the code self-explanatory by using a more explicit variable name (e.g. `screen_reader_only_text`. It's more verbose but we only use it here so it's ok)